### DISCUSSION TITLE
Home app connecting problem

### DISCUSSION THREAD

**Discussion by zhekch** [HomeSpan/HomeSpan#338] [Q&A] [answered]
When I tried to connect my ESP32 to home app, it showed up in accessory list, but I wasn't able to connect. Unlike some other people, MDNS works fine for me, I can view it in the local _hap._tcp. network. I've added the  [video](https://drive.google.com/file/d/1gxhjz_7WsiedpCio2FcHRyQtBim5Twuo/view?usp=sharing), if it can help. Thanks!

**Comment by LuisPaques** [comment]
I have this problem too.

**Comment by HomeSpan** [comment, marked as answer]
@Zhek1ch, I've not seen this problem before.  Based on your video, the Home App is saying the host is unreachable, as opposed to saying the Accessory is invalid.  Can you confirm you are testing with a basic example, such as Tutorial Example 1, without any modifications.  One problem might be the way your router is configured.  Note the ESP32 uses only 2.4 GHz for WiFi.  If your router supports both 2.4 GHz and 5.0 GHz, you need to make sure the router is set to combine the two into a single SSID with cross traffic, else the Home App (which on an iPhone will typically use 5.0 GHz) won't be able to connect to the ESP32.

A few additional tips:  Before pairing, always type 'H' into the CLI.  This ensures the device has a fresh device ID that won't confuse the Home App if you have previously paired with a different set of Accessories.  Also,  set the log mode to Level 2 by typing 'L2' into the CLI before pairing.  This should provide a complete record of all HomeKit network traffic.  During pairing you should see a LOT of activity.  Can you check this?  If you don't see anything once pairing starts, this suggests a WiFi connection issue as described above.

@LuisPaques, can you test the same way as well?  Also can you confirm you are getting the same error - "Device is unreachable"?

**Reply by ZhouMinChuan** [reply]
I still have this problem. Before I try to connect to the device I type 'H' into the monitor, but it is still "unreachable". By the way, even though I type the 'L2', I can't see any message on the monitor.

**Reply by zhekch** [reply]
Typing H, seems to solve the problem, I guess the problem was in me typing F, instead of H, which didn't reset the connections completely, but now it connected perfectly, thanks!
